di_website/publications/models.py

```diff
@@ -1,57 +1,56 @@
-from num2words import num2words
-from itertools import chain
 import operator
 from functools import reduce
+from itertools import chain
 
 from django import forms
-from django.db import models
-from django.db.models import Q, FloatField, Value
-from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
 from django.contrib.contenttypes.models import ContentType
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.db import models
+from django.db.models import FloatField, Q, Value
```

Comment:
This is a duplication ... just keep `from django.db import models`

Reply:
Updated